### PR TITLE
Patch 2

### PR DESCRIPTION
**Pull request by mvibhamenon**
This Pull Request resolves Issue #429 

Description:
In this Pull Request we have done the following additions/edits for the code:

Note: Delete all the text including this!
* Make sure you replace `#<IssueNumber>` with the particular Issue that you are referencing with!
* Make sure you describe the Issue properly and ask as many people as possible to review your Pull Request so that it can be merged as soon as possible!

### CODE REVIEW

**Comment by mvibhamenon** [comment]
Done

File: add.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+a=10
+b=20
+print '(a+b)'
```
Comment (pvgupta24) [inline add.py]: Make changes here
`print (a+b)`

**Comment by salman-bhai** [comment]
Are there any updates or should we close the PR?

**Comment by salman-bhai** [comment]
PR closed due to inactivity!

**Pull request closed without merging**